src/pages/UserPage/UserPage.tsx

```diff
@@ -0,0 +1,99 @@
+import './UserPage.scss';
+import {useEffect} from 'react';
+import {useAppDispatch, useAppSelector} from "../../redux/store";
+import { getDataMainPage } from '../../redux/reducers/pagesDataReducer';
+import { Header } from '../../components/Header/Header';
+import { Footer } from '../../components/Footer/Footer';
+import UserPhoto from '../../assets/img/userImg.png';
+import { IconUserLevel } from '../../assets/icons/icon_user_level';
+import { CommonButton } from '../../components/CommonButton/CommonButton'; 
+import { CustomSlider } from '../../components/CustomSlider/CustomSlider';
+import { TestCard } from '../../components/TestCard/TestCard';
+import { RecomendationCard } from '../../components/RecomendationCard/RecomendationCard';
+
+export const UserPage = () => {
+
+  const dispatch = useAppDispatch()
+  const mainPageData = useAppSelector(state => state.pagesData.mainPageData) 
+
+  useEffect(() => {
+    dispatch(getDataMainPage())
+  }, [])
+
+  return (
+    <div className={"UserPage"}>
+      <Header/>
+      <section className='UserInfo'>
+        <div className='container'>
+          <div className='UserPhoto'>
+            <img src={UserPhoto} alt="userPhoto" />
+          </div>
+          <div className='UserDescription'>
+            <div className='UserDescription-name'>Алена Павлова</div>
+            <ul className='UserDescription-shortResults'>
+              {[1,2,3].map(el => {
+                return <li key={el}>
+                <div className='UserDescription-shortResults-speciality'>QA Manual</div>
+                <div className='UserDescription-shortResults-level'>Уровень прожарки</div>
+                <div className='UserDescription-shortResults-levelIcon'>{IconUserLevel}</div>
```

Comment:
I mean, we can create something like this:
```
const levelText = 'Уровень прожарки';

const userDescription = [
  {
    speciality: 'QA Manual',
    levelText,
    icon: IconUserLevel,
  },
  {
    speciality: 'QA Manual',
    levelText,
    icon: IconUserLevel,
  },
  {
    speciality: 'AQA Pyton',
    levelText,
    icon: IconUserLevel,
  },
];

const UserDescriptions = () => {
  return (
    <ul>
      {userDescription.map((user, index) => (
        <li key={index}>
          <div className='UserDescription-shortResults-speciality'>{user.speciality}</div>
          <div className='UserDescription-shortResults-level'>{user.levelText}</div>
          <div className='UserDescription-shortResults-levelIcon'>{user.icon}</div>
        </li>
      ))}
    </ul>
  );
};

export default UserDescriptions;

```

Reply:
I suggest this data received from backend. Now it is just example

Reply:
Yes, this data will come from the backend and most likely in exactly the form in which I stored it. Therefore, such a structure is completely justified. However, it's up to you.